iceoryx_hoofs/filesystem/source/filesystem.cpp

```diff
@@ -15,7 +15,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "iox/filesystem.hpp"
-#include "iceoryx_hoofs/log/logstream.hpp"
+#include "iox/log/logstream.hpp"
```

Comment:
Okay, now I see why you wanted to have `logstream.hpp` directly in `iox`. I think we can keep is as it is right now since the logstream API needs to be revised anyway.